docs/shared/workspace/grouping-libraries.md

```diff
@@ -0,0 +1,47 @@
+# Grouping Libraries
+
+Libraries should be grouped by _scope_. A library's scope is either application to which it belongs or (for larger applications) a section within that application.
+
+## Example Workspace
+
+Let's use Nrwl Airlines as an example organization. This organization has two apps, `booking` and `check-in`. In the Nx workspace, libraries related to `booking` are grouped under a `libs/booking` folder, libraries related to `check-in` are grouped under a `libs/check-in` folder and libraries used in both applications are placed in `libs/shared`. You can also have nested grouping folders, (i.e. `libs/shared/seatmap`).
+
+The purpose of these folders is to help with organizing by scope. We recommend grouping libraries together which are (usually) updated together. It helps minimize the amount of time a developer spends navigating the folder tree to find the right file.
+
+```txt
```

Comment:
Might be good to mention `@nrwl/angular:move` as a way of moving existing libs around

Reply:
Yes, good point.  I had meant to include that and forgot